capstone/src/main/webapp/script.js

```diff
@@ -241,20 +249,30 @@ function showEdit() {
 
 /** Store edited content from club page */
 function saveClubChanges() {
+  const params = new URLSearchParams(window.location.search);
+
   const newDesc = document.getElementById("description").innerHTML;
   const newWebsite = document.getElementById("website").innerHTML;
-  var newOfficers = [];
 
-  const list = document.getElementById('officers-list');
-  const officerList = list.getElementsByTagName('li');
-  for (var i = 0; i < officerList.length; i++) {
-    newOfficers.push(officerList[i].innerText);
+  var newOfficers = [];
+  const officerListElement = document.getElementById('officers-list');
+  const officerList = officerListElement.getElementsByTagName('li');
+  for (const officer of officerList) {
+    newOfficers.push(officer.innerText);
   }
   
+  var newLabels = [];
+  const labelsListElement = document.getElementById('labels');
+  const labelsList = labelsListElement.getElementsByTagName('li');
+  for (const label of labelsList) {
+    newLabels.push(label.innerText.replace(/\s+/g, '')); // Get rid of whitespaces
```

Comment:
I thought labels are already sanitized at this point? It's possible to still have whitespace here but not uppercase?

Reply:
This was a workaround for a strange behavior I was seeing, where leading and trailing (or multiple in a row) whitespaces wouldn't be filtered out properly by the Java servlet. After a lot of debugging, I finally decided to just strip whitespaces in the JS code as well.